### PR TITLE
FE-1287 - UX feedback for Pinned Report 

### PR DESCRIPTION
**Pull request by shrirupa**
FE-1287 - UX feedback for Pinned Report 

### What Changed
 
- Added a `small` prop for chart components so that height if a single chart is 200 on dashboard for pinned report
- Radio button are now the part of Name columns in Change Report Modal
- Made the Change Report Modal accessible by keyboard (I can  give a walkthrough for the change here) 
- Saved Report Modal on Analytics Report - Action is always visible to Schedule and Pin, hide Edit and Delete based on whether user can edit or not

### How To Test

- On Dashboard, when the pinned report has single chart, max height should be more than 200.
- On Dashboard, Radio buttons in Change Report Modal are now part of Name Column
- On Dashboard, Radio buttons on Change Report Modal are accessible by keyboard
- On Analytics Report Page, Saved Report Modal - Action is now always visible with Schedule and Pin options, and Edit and Delete are hidden based on whether user can edit or not

### To Do

- [ ] Address feedback

### CODE REVIEW

File: cypress/tests/integration/dashboardV2/dashboardPageV2.spec.js [hunk 1]
```diff
@@ -66,7 +66,10 @@ describe('Version 2 of the dashboard page', () => {
         requestAlias: 'updateUIOptions',
       });
       cy.withinModal(() => {
-        cy.findByLabelText('Report d50d8475-d4e8-4df0-950f-b142f77df0bf').check({ force: true });
+        // eslint-disable-next-line cypress/no-unnecessary-waiting
+        cy.wait(1000);
```
Comment (gfriedmansp) [inline cypress/tests/integration/dashboardV2/dashboardPageV2.spec.js:70]: Could we get a comment in here as to _why_ we're doing this when we don't usually want to?

File: src/pages/dashboardV2/components/ChangeReportModal.js [hunk 2]
```diff
@@ -77,25 +63,37 @@ export function ChangeReportModal({ reports, open, onClose, currentUser }) {
...
-          Cancel
-        </Button>
-      </Modal.Footer>
+      <ModalFooter onClose={closeModal} control={control} />
     </Modal>
   );
 }
 
+function ModalFooter({ onClose, control }) {
+  const submitDisbaled = useWatch({
```
Comment (gfriedmansp) [inline src/pages/dashboardV2/components/ChangeReportModal.js:74]: Haven't seen `useWatch` before. Looks cool  👍

Comment (shrirupa) [reply src/pages/dashboardV2/components/ChangeReportModal.js:74]: yeah, its really cool, localizes re rendering to the component only

**Review by gfriedmansp — APPROVED** [review]
I think all looks good here. Seems to be working as expected. Let me know when you want to deploy it.

**Review by gfriedmansp — APPROVED** [review]
LGTM with the error checking in onSubmit

**Pull request merged**